adding_species_manuscript.md

```diff
@@ -182,13 +185,36 @@ Users can also add other attributes of a species to the simulation that are not
 
         - eg. "often reported parameter values are tied to a model and its other parameters, so mixing values from different sources has its own caveats" (https://github.com/popsim-consortium/adding-species-manuscript/issues/7)
 
-    - what about species that don't have some of those resources? **PETER**
 
-        - if everything is available except the genome is contig-level (and no recombination map):
-
-            -   would want to choose just a few of the most important or longest contigs to simulate
-            -   https://github.com/popsim-consortium/adding-species-manuscript/issues/1#issuecomment-1050081855
-            -   might be more appropriate to just simulate a locus of appropriate length
+## What if we don't know everything?
+
+When we set out to cast a wide net and add a wide variety of species to the catalogue,
+we quickly ran into species that people were enthusiastic to add,
+but lacked many (or most) of the ingredients above.
+When is this important?
+The utility of stdpopsim is to make tricky data easily available for simulation;
+such data might be genetic maps, annotations, and/or demographic models.
+We have not yet encountered a species with widely-used demographic models but no chromosome-level assembly,
+so the main issue in practice seems to be around chromosome-level assemblies.
+To consider this, it helps to consider the use cases for stdpopsim.
+If there is not a chromosome-level assembly,
+then we need to either just simulate many, small chunks of genome,
+or simulate an anonymous chromosome, not specific to the species.
+We could treat each of the many contigs of an assembly as a "chromosome" in stdpopsim.
+However,
+this is not right because it's missing the effects of linked selection (although it may be useful),
+but more crucially, it would be a significant amount of effort for stdpopsim developers
+that wouldn't lower barriers for downstream users much at all.
+With a great many contigs,
```

Comment:
I think we could make this section more general by using this paragraph (without the reference to stdpopsim), and a little bit about what you'll be missing (linkage, so selection analyses won't be as good as they would otherwise be). Then the stuff above this about stdpopsim development etc. could go into section 5 (examples from hackathon/what we learned).

Reply:
Yes, good point. I've had a go at this, but more could be done.